pex/bin/pex.py

```diff
@@ -159,7 +159,7 @@ def configure_clp_pex_resolution(parser, builder):
   group.add_option(
       '--cache-dir',
       dest='cache_dir',
-      default=os.path.expanduser('~/.pex/build'),
+      default='{pex_root}/build',
```

Comment:
I think you're misunderstanding my original ask. it seems like setting a default value for `--pex-root` doesn't really change anything here and would obviate the `if options.pex_root:` check below. can you reconcile that if you decide to continue down this route? otherwise I think this was better as it was before.

regarding the defaults strings - this definitely needs to be made clearer. at a minimum tho, we should have sane values for the 99% case where folks aren't passing `--pex_root` that doesn't require users to understand how `.format` strings work and apply them in a contextless situation.

how about including the /literal/ default in each options /help string/ along with a note about how modifying pex_root will also affect the default value?